ORM/ORMFundamentals/Services/OrderService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using ORMFundamentals.Entities;
+using ORMFundamentals.Repositories;
+
+namespace ORMFundamentals.Services;
+
+public class OrderService : IOrderService
+{
+    private readonly IOrderRepository _orderRepository;
+
+    public OrderService(IOrderRepository orderRepository)
+    {
+        _orderRepository = orderRepository;
+    }
+
+    public async Task<IEnumerable<Order>> GetAll()
+    {
+        return await _orderRepository.GetAllOrders();
+    }
+
+    public async Task<Order> GetOrderById(int id)
+    {
+        return await _orderRepository.GetOrderById(id);
+    }
+
+    public async Task Add(Order order)
+    {
+        await _orderRepository.AddOrder(order);
+    }
+
+    public async Task Update(Order order)
+    {
+        await _orderRepository.UpdateOrder(order);
+    }
+
+    public async Task Delete(int id)
+    {
+        await _orderRepository.DeleteOrder(id);
+    }
+
+    public async Task<IEnumerable<Order>> GetOrdersByMonth(int month)
+    {
+        return await _orderRepository.GetOrdersByCondition(o => o.CreatedDate.Month == month);
```

Comment:
Don't forget to use full name argument for lambda expression. This is especially important if your method accepts several overloading with different argument types.